tests/scripts/build-airgap

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+# Build Airgap
+
+set -e -x
+
+K3S_UPSTREAM_VERSION=$1
+CERT_MANAGER_VERSION=$2
+RANCHER_CHANNEL=$3
+K3S_DOWNSTREAM_VERSION=$4
+
+# Create directories
+mkdir -p /opt/rancher/{k3s_$K3S_UPSTREAM_VERSION,helm} /opt/rancher/images/{cert,rancher,registry,elemental}
+cd /opt/rancher/k3s_$K3S_UPSTREAM_VERSION/
+
+# Install packages
+zypper --no-refresh -n in zstd skopeo yq
+
+# Add rancher manager in /etc/hosts
+sudo echo "192.168.122.102 rancher-manager.test" >> /etc/hosts
+
+# Download k3s and rancher
+curl -#OL https://github.com/k3s-io/k3s/releases/download/v$K3S_UPSTREAM_VERSION%2Bk3s1/k3s-airgap-images-amd64.tar.zst
+curl -#OL https://github.com/k3s-io/k3s/releases/download/v$K3S_UPSTREAM_VERSION%2Bk3s1/k3s
+
+# Get the install script
+curl -sfL https://get.k3s.io/ -o install.sh
+
+# Get the airgap deploy script
+cp /home/gh-runner/actions-runner/_work/elemental/elemental/tests/scripts/deploy-airgap . 
+
+# Get Helm Charts
+cd /opt/rancher/helm/
+
+# Add repos
+helm repo add jetstack https://charts.jetstack.io > /dev/null 2>&1
+#helm repo add rancher-latest https://releases.rancher.com/server-charts/latest > /dev/null 2>&1
+helm repo add rancher-$RANCHER_CHANNEL https://releases.rancher.com/server-charts/$RANCHER_CHANNEL > /dev/null 2>&1
+helm repo update > /dev/null 2>&1
+
+# Get charts
+helm pull jetstack/cert-manager --version $CERT_MANAGER_VERSION > /dev/null 2>&1
+#helm pull rancher-latest/rancher --version v$RANCHER_MANAGER_VERSION > /dev/null 2>&1
+if [ "$RANCHER_CHANNEL" == "latest" ]; then
```

Comment:
If you use `==` you have to use `[[ .. ]]` for the test and not `[ ...]` (and `[[ ... ]]` is better anyway).